src/Network/Network.Test/ScenarioTests/VirtualNetworkTests.ps1

```diff
@@ -245,6 +245,54 @@ function Test-InitializeSubnetPolicy
     }
 }
 
+<#
+.SYNOPSIS
+Tests resetting subnet Policies on virtualNetwork for a delegated service name.
+.DESCRIPTION
+SmokeTest
+#>
+function Test-ResetAzVirtualNetworkSubnetPolicy
```

Comment:
I don't see a json recording for this test - did you forget to check it in?  This will cause CI tests to fail.

Reply:
service changes are not yet deployed to all regions will update recordings after they are available.